sdk/schemaregistry/Azure.Data.SchemaRegistry/tests/SchemaRegistryClientLiveTests.cs

```diff
@@ -23,27 +24,43 @@ public SchemaRegistryClientLiveTests(bool isAsync, SchemaRegistryClientOptions.S
             _serviceVersion = version;
         }
 
-        private SchemaRegistryClient CreateClient(string format)
+        private string GetEndpoint(string format, int group)
         {
-            string endpoint;
-            switch (format)
+            switch (format, group)
             {
-                case Json:
-                    endpoint = TestEnvironment.SchemaRegistryEndpointJson;
-                    break;
-                case Avro:
-                    endpoint = TestEnvironment.SchemaRegistryEndpointAvro;
-                    break;
-                case Custom:
-                    endpoint = TestEnvironment.SchemaRegistryEndpointCustom;
-                    break;
+                case (Avro, 1):
+                    return TestEnvironment.SchemaRegistryEndpointAvro1;
```

Comment:
I don't quite understand how we went from having one Avro endpoint to needing 4 just from adding the new service version. Are we really creating more than 25 schemas per service version?

Reply:
Yeah, I'm honestly not sure either, I tried just adding one but then was seeing local failures when running the tests in live mode. But also when I run them locally I have 279 tests, between the three .NET versions, the different API versions, and async/sync

Reply:
I would focus on one platform. Sure the local experience won't be great, but at least for the live test pipeline it will work.

Reply:
Do the live test pipelines only run one version of .NET? If so, then I think we could just go with one for each service version

Reply:
Yeah I can look into adding that, I have been deleting them in the portal in between runs (unfortunately, generating new test resources doesn't actually delete the schemas :/ )

Reply:
> Another thing we could do would be to automatically delete the schema groups at the end of each test run so that we can make things work locally.

I think for now it's fine, but I can add it in the future if it gets annoying